src/components/InputGroup/InputGroup.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './style.css';
+
+
+// eslint-disable-next-line object-curly-newline
+const InputGroup = ({ label, type, name, placeholder, onChangeFunc, onBlurFunc, value, check, status, isRequired }) => <>
```

Comment:
when we have more than 3 props it's a good practice to put each of them in new line:
```
const InputGroup = ({
  label, 
  type, 
  name, 
  placeholder, 
  onChangeFunc, 
  onBlurFunc, 
  value, 
  check, 
  status, 
  isRequired 
}) => (
//your code
)
```